DateCalculator/DateCalculator.py

```diff
@@ -73,45 +79,56 @@ def build_gui(self):
         self.top = Gtk.Box(orientation=Gtk.Orientation.VERTICAL)
         self.top.set_border_width(6)
 
-        self.entry1 = self.__add_text_view(_("Expression 1"))
-        self.entry2 = self.__add_text_view(_("Expression 2"))
-        self.result = self.__add_text_view(_("Result"))
+        self.entry1 = self.__add_entry(
+            _("Expression 1: Reference Date or Date Range"), _("a valid Gramps date")
```

Comment:
Does "Expression 1" (and 2) help any more? With the proper description of the input field, what value does it add?

Reply:
Two of the error messages still refer to "expression 1" or "expression 2".
If they are reworded then I agree it is clearer to remove "Expression 1:" and "Expression 2:" from the labels.

I've slightly reworded the relevant error messages and removed the label prefix.
I think the error messages could still be improved - ideas welcome!

Reply:
I see your dilemma. Ideally Gramps UI would have field validation and in-place error indication like a red border on error or a tooltip indicating the error. Without that the error message has to repeat the field description to be completely accurate. Thanks for making the change.